### PR TITLE
feat: Add watch command

### PR DESCRIPTION
**Pull request by lalizita**
This PR builds the watch command

- [x] Watch files 
- [x]  Run exercises when save a file
- [x]  Read List command
- [x]  Read Hint Command
- [x]  Clear the terminal on every command

### CODE REVIEW

File: golings/cmd/watch.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+func WatchEvents(updateF chan<- string) {
+	log.Println("===========WATCHER")
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer watcher.Close()
+
+	path, _ := os.Getwd()
+	file_path := fmt.Sprintf("%s/exercises", path)
```
Comment (mauricioabreu) [inline golings/cmd/watch.go:87]: This variable should be camelCase, like `filePath` except when you want them to be exported

**Review by mauricioabreu — CHANGES_REQUESTED** [review]

File: golings/cmd/watch.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		Use:   "watch",
+		Short: "Run a single exercise",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			RunNextExercise(infoFile)
+			reader := bufio.NewReader(os.Stdin)
+			update := make(chan string)
+
+			go WatchEvents(update)
+			go func() {
+				for {
```
[the remark this answers is not in the document]
Comment (lalizita) [reply golings/cmd/watch.go]: good!

File: golings/cmd/watch.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+	}
+}
+
+func WatchEvents(updateF chan<- string) {
+	log.Println("===========WATCHER")
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer watcher.Close()
```
[the remark this answers is not in the document]
Comment (lalizita) [reply golings/cmd/watch.go]: I don't understand why I must not close the watcher, Could it make a memory lock problem happens?

**Review by mauricioabreu — APPROVED** [review]
Awesome! 🥇

**Comment by mauricioabreu** [comment]
@lalizita want me to merge this?

**Comment by lalizita** [comment]
yeah @mauricioabreu , should you merge it please?

**Pull request merged**